server/auth/types/openid/openid_auth.ts

```diff
@@ -60,6 +62,9 @@ export class OpenIdAuthentication extends AuthenticationType {
   private openIdConnectUrl: string;
   private wreckClient: typeof wreck;
 
+  private readonly extraCookiePrefix: string = '';
```

Comment:
That is the cookie prefix that is used by the additional cookies.
I had this before the number of cookies was configurable, and the cookie prefix was the only thing you could configure.

But thanks for pointing this out - since we read the config on multiple places now when interacting with the new cookies, I don't think we need this instance variable. I'll refactor this away.